drivers/SmartThings/matter-lock/src/test/test_matter_lock_batteryLevel.lua

```diff
@@ -0,0 +1,89 @@
+-- Copyright 2022 SmartThings
```

Comment:
Super nit: stop living in the past, it's 2024!

Reply:
done!